test/e2e/framework/kubelet_stats.go

```diff
@@ -97,10 +97,13 @@ func getKubeletMetrics(c clientset.Interface, nodeName string) (metrics.KubeletM
 	return kubeletMetrics, nil
 }
 
-// GetKubeletLatencyMetrics gets all latency related kubelet metrics. Note that the KubeletMetrcis
-// passed in should not contain subsystem prefix.
-func GetKubeletLatencyMetrics(ms metrics.KubeletMetrics) KubeletLatencyMetrics {
-	latencyMethods := sets.NewString(
+// GetDefaultKubeletLatencyMetrics calls GetKubeletLatencyMetrics with a set of default metricNames
+// identifying common latency metrics.
+// Note that the KubeletMetrics passed in should not contain subsystem prefix.
+// TODO(mtaufen): It looks like some latency metrics have been added since latencyMethods in this function was updated,
```

Comment:
I'd like to resolve whether we need to update this before merging the PR.

Reply:
I don't think we need to.  I think this is just the list of metrics used by e2e tests, not necessarily all metrics.

Reply:
Ok, I'll remove the TODO then.